### PR TITLE
Adding support for filtering specific proto files 

### PR DESCRIPTION
**Pull request by patrikhuss**
We had a case where the source directory had a ton of proto files, and to reduce unused code we only wanted to compile a few of them @fylke @patrikhuss

### CODE REVIEW

File: src/rebar3_gpb_compiler.erl [hunk 1]
```diff
@@ -36,23 +38,30 @@ compile(AppInfo, State) ->
...
+                     filter_unwanted_protos(WantedProtos, FoundProtos)
+             end,
+    rebar_api:debug("Filtered protos: ~p", [Protos]),
     %% set the full path for the output directories
     %% add to include path dir locations of the protos
     %% remove the plugin specific options since gpb will not understand them
     GpbOpts = remove_plugin_opts(
                 proto_include_paths(AppDir, Protos,
-                  default_include_opts(AppDir, DepsDir,
+                                    default_include_opts(AppDir, DepsDir,
```
[the remark this answers is not in the document]
Comment (patrikhuss) [reply src/rebar3_gpb_compiler.erl]: My edts corrected indentation here "to edts standard :-)",   will revert this

File: src/rebar3_gpb_compiler.erl [hunk 2]
```diff
@@ -152,13 +161,37 @@ filter_included_proto(Source, Dep, GpbOpts) ->
                                              GpbOpts ++ [to_proto_defs, return]),
     Imports = lists:map(fun(Import0) ->
                           {ok, Import} = gpb_compile:locate_import(Import0, GpbOpts),
-                          filename:absname(Import)
+                                filename:absname(Import)
```
[the remark this answers is not in the document]
Comment (patrikhuss) [reply src/rebar3_gpb_compiler.erl]: fixed

**Comment by lrascao** [comment]
thanks!

**Comment by lrascao** [comment]
`2.10.0` pushed to hex

**Pull request merged**